### ISSUE TITLE
TreeTagger importer mangles document names with dot

### ISSUE THREAD

**Issue by amir-zeldes** [korpling/treetagger-emf-api#1]
If a file is named `my.file.tt`, the name of the imported document in Salt becomes just `my`, instead of `my.file`. I think this line is the problem:

```
String documentBaseName = this.getURI().lastSegment().split("[.]")[0];
```
See:

https://github.com/korpling/treetagger-emf-api/blob/37ca35e4e84b4749b7666fae518810719424b9bf/src/main/java/de/hu_berlin/german/korpling/saltnpepper/misc/treetagger/resources/TabResource.java#L494

**Comment by amir-zeldes** [comment]
@FlorianZipser @thomaskrause - I don't want to mess anything up, if you can have a look I think it should be easy to fix?

**Comment by amir-zeldes** [comment]
This seems to work for me locally, see: 1cef574358537395ded6770695ef5e8f1b4b32d4

**Comment by thomaskrause** [comment]
I think the change is semantically ok, since only the part after the last dot should be considered a file extension. However, I can't compile the code because the ```location''' variable does not exist. Has there been any other change this commit depends on?

**Comment by amir-zeldes** [comment]
That's weird... in order for it to work, I just edited it inside the dependency in TreeTagger-Modules, and copy pasted the code here because I figured the source was identical. Lesson for me: This is what happens when you commit directly on GitHub out of laziness... 

I now realize the file I was editing was TabReader.java in org.corpus_tools.peppermodules.treetagger.model.resources. On line 93 it says:

```
	private URI location = null;
```

But I can't find TabReader.java anywhere in the korpling organization on github. It's somehow not in the TT-Modules repo either. Any idea where I should have committed this?
